DomainTools/python_api
# setting up a min limit default value

how to test it:

1. Go to file api.py (../python_api/domaintools/api.py) : and change the URL `https://api.domaintools.com` to the URL where the API changes have been deployed
2. on root folder do pip install .
3. run python (or python3)
4. inside the command shell run in this order:

```
import domaintools
a = domaintools.API('api_username', 'api_key')
result = a.account_information().data()
from pprint import pprint 
pprint(result)

a._rate_limit()
```

It should throw no errors
